### PR TITLE
Eddie Leung - WhoAreWe

### LINKED ISSUE #67: WhoWeAre component
image

`src/components/about/WhoWeAre.jsx`

Image is in `public/images/about/JMbrochures.webp`

Use the existing Header.jsx component

### LINKED ISSUE #77: Responsify WhoWeAre
image

- Make it look good on mobile like the Figma.
- When you are done, on your PR, upload a screenshot of the component on mobile AND laptop/desktop

### PR DESCRIPTION
**Pull request by Ezkron**
Heres what it looks like currently.

image

### CODE REVIEW

**Comment by Ezkron** [comment]
Just need to fix the formatting in order to fit the image

File: src/components/about/WhoWeAre.jsx [hunk 1]
```diff
@@ -1,7 +1,29 @@
 import React from "react";
+import Header from "./Header";
 
 const WhoWeAre = () => {
-  return <div>WhoWeAre</div>;
+  return (
+    <div className="p-6">
```
Comment (SafeDuck) [inline src/components/about/WhoWeAre.jsx]: use w-3/4.
use flex, flex-col, gap-10

**Review by SafeDuck — CHANGES_REQUESTED** [review]

**Comment by Ezkron** [comment]
After making the changes, here is what it looks like now

image

File: src/components/about/WhoWeAre.jsx [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
+          system applies to all aspects of the company. Our goal is to deliver
+          defect-free product on-time at the most competitive price possible.
+        </p>
+        <div className="text-left mb-8">
+          <p className="font-hk bg-gradient-to-r from-jm-blue-300 to-jm-blue-100 bg-clip-text text-transparent ${size} tracking-widest font-bold">
+            Brochures
+          </p>
+        </div>
+        <div className="flex justify-center"></div>
+        <img
```
Comment (SafeDuck) [inline src/components/about/WhoWeAre.jsx]: Use Image component from next.

File: src/components/about/WhoWeAre.jsx [hunk 3]
```diff
@@ -1,7 +1,32 @@
...
+          src="/images/about/JMbrochures.webp"
```
Comment (SafeDuck) [inline src/components/about/WhoWeAre.jsx]: Statically import the image at the top, instead of putting the path directly inside the Image

**Review by SafeDuck — CHANGES_REQUESTED** [review]

**Comment by Ezkron** [comment]
image

**Pull request closed without merging**